editor/gui/editor_quick_open_dialog.h

```diff
@@ -97,6 +97,9 @@ class QuickOpenResultContainer : public VBoxContainer {
 	bool has_nothing_selected() const;
 	String get_selected() const;
 
+	bool get_instant_preview_enabled() const;
```

Comment:
```suggestion
	bool is_instant_preview_enabled() const;
```
Also the usual order is setter, getter, so this method should be under `set_instant_preview_toggle_visible()`.